### PR TITLE
Make silk default tool for inspecting performance #8159

### PR DESCRIPTION
**Pull request by chiatt**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Replaces debug_toolbar with silk as the default tool for performance analysis

### Issues Solved

# 8159

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: arches/install/arches-templates/project_name/settings.py-tpl [hunk 1]
```diff
@@ -74,26 +74,44 @@ DATABASES = {
...
+    "django.middleware.locale.LocaleMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "arches.app.utils.middleware.ModifyAuthorizationHeader",
+    "oauth2_provider.middleware.OAuth2TokenMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    # "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "arches.app.utils.middleware.SetAnonymousUser",
+    # "silk.middleware.SilkyMiddleware",
```
Comment (chrabyrd) [inline arches/install/arches-templates/project_name/settings.py-tpl:112]: I am unable to get silk running with `silk.middleware.SilkyMiddleware`, but I'm able to get it running with `silk`

**Review by chrabyrd — COMMENTED** [review]
overall looks good! Just one question/comment 👍

File: arches/install/arches-templates/project_name/settings.py-tpl (hunk 1 above)
Comment (chiatt) [reply arches/install/arches-templates/project_name/settings.py-tpl:112]: If I change the `MIDDLEWARE` to `silk` things break, but changing `INSTALLED_APPS` from `silk.apps.SilkAppConfig` to `silk` works okay (and is now consistent with the docs). Can you give it another try with that change?

**Review by chrabyrd — APPROVED** [review]
lgtm!

**Pull request merged**